### PR TITLE
Logging: Track group unit counts for each machine network owner.

### PR DESCRIPTION
**Pull request by dijksterhuis**
TL;DR -- track who is a group owner of units based on the [Machine Network ID](https://community.bistudio.com/wiki/Multiplayer_Scripting#Machine_network_ID) of the owner of the group

- server will be ID `2`
- everyone else will be a number higher than `2`

anyone who has created a unit (some created object or zeus'd AI) will have a number greater than 1 in the output

this is what it looks like without filtering for numbers > 1 .... I am ID number `24` because I zeus'd in in 1x AI -- all players have 1x `unit` ... themselves
```
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=24 count=2"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=9 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=19 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=22 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=25 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=11 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=15 count=3"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=13 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=20 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=23 count=1"
16:58:53 "2024-7-5 21:58:53:785 | MIKEFORCE | INFO | File:  |  | Group owners: ownerMachineNetId=2 count=107"
```

Going to filter out all numbers == 1 so we avoid spamming logs.

----

TODO

- [ ] Can we map machine network IDs back to player IDs?
- [ ]  Can we aggregate unit counts by the kind of object? player/AI/vehicle etc

### CODE REVIEW

**Comment by dijksterhuis** [comment]
closing -- not doing
